batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/validator/GenericValidator.java

```diff
@@ -1,5 +1,7 @@
 package com.lftechnology.batch7crud.validator;
 
+import com.lftechnology.batch7crud.exception.ValidationException;
+
 import java.util.Map;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Map'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)